saskatoon/harvest/views.py

```diff
@@ -132,6 +133,18 @@ def get_context_data(self, **kwargs):
         context['cancel_url'] = reverse_lazy('harvest-detail', kwargs={'pk': self.object.pk})
         return context
 
+    def get_success_message(self, cleaned_data) -> str:
+        pick_leader_harvest_number: int = len(
+            self.object.pick_leader.person.harvests_as_pickleader
```

Comment:
Is this list contains harvests from past seasons ?

Reply:
I believe so yes, all the harvests the person has done. @tristanlatr 
```python
class Person:
    ...

    @property
    def harvests_as_pickleader(self):
        return Harvest.objects.filter(pick_leader=self.auth_user)
```